src/components/Happy.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+
+class Happy extends Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            currentEnthusiasm : 1,
+            value: ''
+        }
+    };
+
+    updateEnthusiasm = (currentEnthusiasm) => this.setState({currentEnthusiasm});
+    onPlus = () => this.updateEnthusiasm(this.state.currentEnthusiasm + 1);
```

Comment:
aici si la urmatorul rand vrei sa folosesti valoarea curenta a currentEnthusiasm din state, better daca folosesti
`this.setState(prevState => ({ currentEnthusiasm: prevState.currentEnthusiasm + 1 }))`